spec/features/users/regular_user_show_spec.rb

```diff
@@ -46,4 +47,23 @@
     expect(page).to_not have_content(@regular_user.name)
     expect(page).to have_content("Changes Made to Profile Successfully")
   end
+
+  it "has link to edit password and ability to change password" do
+    click_link "Edit Password"
```

Comment:
Add spec to edit password and make sure that the password and password confirmation match